### DISCUSSION TITLE
2D osu!catch

### DISCUSSION THREAD

**Discussion by rasputozen** [ppy/osu#15201] [Ideas]
Exactly like normal osu!catch but the circles are mirrored on the perpendicular axis and you use the up/down arrows along with left/right to go to where the mirrored circles converge. Because the circles will converge at various places on the screen instead of just the bottom in normal osu!catch the speed of the mirror circle pairs needs to be scaled up or down in proportion to the distance the circles must travel from the spawn point to the point of convergence.

**Comment by rasputozen** [comment]
Actually I think I just invented normal osu but you control cursor with the arrow keys instead of a mouse lol. There's no point to have converging circles, you could just have the normal osu approach circles and remove the need to click. As long as the cursor is brought within the circle at correct point of rhythm it would count as a hit like in osu!catch.

**Comment by smoogipoo** [comment]
I don't understand the explanation, can you provide a diagram of some sort?

**Reply by rasputozen** [reply]
I realize it would actually just be regular osu but using the arrow keys instead of a mouse lol.
